src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/HashCodeOnlyCallsSuper.xml

```diff
@@ -145,4 +145,22 @@ class EqHashTryout2 {
      ]]></code>
     </test-code>
 
+    <test-code>
+        <description>no violation: Issue false positive for HashCodeOnlyCallsSuper -- not when equals also calls only super #483 </description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+class Foo {
+    @Override
+    public boolean equals(Object o) {
```

Comment:
might add a case where equals contains another line, e.g. a valid issue..?

Can also be an enum compare, e.g. `status == o.status && super.equals(o)`?

Reply:
Good point